app/assets/javascripts/components/mentee_application_step_3.js.coffee

```diff
@@ -21,15 +37,22 @@
       React.createElement(ApplicationRadioGroup, key: 'previousProgrammingExperience', field: 'previous_programming_experience', options: @yesNoOptions(), placeholder: "Do you have previous experience in any other programming language?", application: @props.application, setApplicationField: @props.setApplicationField)
       if @props.application.previous_programming_experience == 'true'
         React.createElement(ApplicationTextArea, key: 'experience', field: 'experience', label: "Please name the language(s), your level of expertise (beginner, intermediate, professional) and tell us about the projects you have worked on.", application: @props.application, setApplicationField: @props.setApplicationField)
       React.createElement(ApplicationRadioGroup, key: 'operatingSystem', field: 'operating_system', options: @operating_systems(), placeholder: "What operation system are you using?", application: @props.application, setApplicationField: @props.setApplicationField)
       if @props.application.programming_language == 'ruby' && @props.application.operating_system == 'windows'
         @systemWarning()
       React.createElement(ApplicationTextArea, key: 'projectProposal', field: 'project_proposal', label: "What project would you like to develop during the learning period? Please come up with your own idea", application: @props.application, setApplicationField: @props.setApplicationField)
-      React.createElement(ApplicationTextArea, key: 'roadmap', field: 'roadmap', label: "If you were to implement the application described in the previous question, what would be your roadmap?", application: @props.application, setApplicationField: @props.setApplicationField)
+      React.createElement(ApplicationTextArea, key: 'roadmap', field: 'roadmap', label: "Imagine you have been accepted to the program. Please describe how a typical week as a mentee at Learn IT, Girl! looks like for you.", application: @props.application, setApplicationField: @props.setApplicationField)
       React.createElement(ApplicationRadioGroup, key: 'timeAvailability', field: 'time_availability', options: @props.time_availabilities, placeholder: "How much time can you dedicate to learning through the program?", application: @props.application, setApplicationField: @props.setApplicationField)
       React.createElement(ApplicationCheckboxGroup, key: 'engagements', field: 'engagements', options: @props.engagements, placeholder: "Do you have any other engagements during the 3-month program period? Check all that apply", application: @props.application, setApplicationField: @props.setApplicationField)
```

Comment:
BUG:
In last page:
"check all that apply": I can only select one option

Reply:
Fixed